static/main.js

```diff
@@ -0,0 +1,87 @@
+$(document).ready(function () {
+  $('.add_entry').on('submit', function(event){
+      event.preventDefault();
+      add_post();
+    });
+});
+
+
+
+
+function add_post() {
+    var title = $('#title').val();
+    var text = $('#text').val();
+    $.ajax({
+      url: '/add',
+      type: 'POST',
+      dataType: 'json',
+      data: {'title': title, 'text': text},
+      success: success
+    });
+}
+
+function open_edit() {
+    var id = $('.entry').attr('id').split("entry")[1];
+    $.ajax({
+      url: '/editview',
+      type: 'GET',
+      dataType: 'json',
+      data: {'id': id},
+      success: open_edit_success,
+    });
+}
+
+function edit_post() {
+    var title = $('#title').val();
+    var text = $('#text').val();
+    var split_path = window.location.pathname.split("/");
+    var id = split_path[split_path.length-1];
```

Comment:
This is "tight coupling", meaning that your javascript is dependent on an implementation detail of your website backend.  That's not a great situation.  It would be better to get the id from the form.  HTML supports a `hidden` input type, which is ideal for this type of situation.  If you have one in your form, you can put the ID into the form without showing it to the user, or allowing them to edit it.